src/Uno.UI/UI/Xaml/Controls/Primitives/SelectorItem.cs

```diff
@@ -186,7 +187,7 @@ private void UpdateCommonStates(ManipulationUpdateKind manipulationUpdate = Mani
 			{
 				CoreDispatcher.Main.RunAsync(CoreDispatcherPriority.Normal, async () =>
 				{
-					await Task.Delay(delay);
+					await Task.Delay(ToTimeSpan(delay));
```

Comment:
you can using [Task.Delay(int millisecondsDelay)](https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.task.delay?view=net-6.0#system-threading-tasks-task-delay(system-int32)) to avoid convert to TimeSpan

Reply:
In this case, it's not possible because of the time resolution.